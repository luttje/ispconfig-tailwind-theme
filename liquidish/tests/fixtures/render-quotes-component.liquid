<input type="text"
    value="{{ value }}"
    class="w-full border rounded px-2 py-1"
    onkeyup="{{ onkeyup }}" />
