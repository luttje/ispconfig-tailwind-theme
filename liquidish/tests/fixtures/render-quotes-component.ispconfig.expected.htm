<input type="text"
    value="{tmpl_var name="value"}"
    class="w-full border rounded px-2 py-1"
    onkeyup="pass_check(this.value);checkPassMatch('password','repeat_password');" />
